packages/entity-browser/src/entitybrowser.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {withKnobs} from '@storybook/addon-knobs'
+
+import EntityBrowserApp from './main'
+
+storiesOf('Entity Browser', module)
+  .addDecorator(withKnobs)
+  .add(
+    'Entity Browser',
```

Comment:
list parameter list of Entity-Browser is incomplete. An please add them as knobs when reasonable.